File: webview/src/main/java/cn/gd/snm/testwebview/MainActivity.java
package cn.gd.snm.testwebview;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.webkit.WebViewClient;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);


        setContentView(R.layout.activity_main);

        findViewById(R.id.bt).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, WebviewActivity.class));
            }
        });
    }
}

File: glide-4-8-0/src/main/java/cn/gd/snm/glide_4_8_0/MainActivity.kt
package cn.gd.snm.glide_4_8_0

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.bumptech.glide.Glide
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        test()
    }

    private val gifUrl = "https://img.zcool.cn/community/01c5695a52dac5a8012180c527476b.gif"
    private fun test() {
        Glide.with(this).load(gifUrl).into(img)
    }
}